.github/workflows/helmsman_mosip.yml

```
@@ -63,6 +71,24 @@ jobs:
             echo "HELMSMAN_MODE=${{ github.event.inputs.mode }}" >> $GITHUB_ENV
           fi        
 
+      - name: Set Profile
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:13:30: Double quote to prevent globbing and word splitting

(shellcheck)